include/rgf/README.md

```diff
@@ -0,0 +1,168 @@
+# RGF Version 1.3
+
+### C++ Programs for Regularized Greedy Forest (RGF)
+
+************************************************************************
+
+# 0.0 Changes
+
+## March 2018 (version 1.3)
+
+  1. Calculation of feature importance has been added.
+
+  2. Dumping information about the forest model to the console has been added.
+
+  3. License has been changed from GPLv3 to MIT.
+
+## February 2018
+
+  1. Absolute error loss has been added.
+
+## August-December 2017
+
+  1. The executable file for 32-bit Windows has been added.
+
+  2. `CMakeLists.txt` file has been added.
+
+  3. Compilation on 32-bit Windows has been fixed.
+     Also, compilation with newer MS Visual Studios has been added.
+
+## June 2014
+
+  1. The restriction on the size of training data files has been removed.
+ 
+     _Old_: Training data files had to be smaller than 2GB.
+
+     _New_: No restriction on the training data file sizes.
+            (However, the number of lines and the length of each line must
+            be smaller than 2^31 (2,147,483,648).)
+
+  2. The solution file for MS Visual C++ 2010 Express has been changed
+     from 32-bit to 64-bit; also, `__AZ_MSDN__` has been added to
+     Preprocessor Definitions.
+
+  3. Some source code files have been changed.
+
+## September 2012 (version 1.2)
+
+  1. The initial release.
+
+# 0.1 Acknowledgements
+
+  We thank **Dave Slate** for suggesting a solution to the issue of file size
+  restriction above.
+
+************************************************************************
+
+# Contents:
+
+1. [Introduction](#1-introduction)
+
+1.1. [System Requirements](#11-system-requirements)
+
+2. [Download and Installation](#2-download-and-installation)
+
+3. [Creating the Executable](#3-creating-the-executable)
+
+3.1. [Windows](#31-windows)
+
+3.2. [Unix-like Systems](#32-unix-like-systems)
+
+3.3. [[Optional] Endianness Consideration](#33-optional-endianness-consideration)
+
+4. [Documentation](#4-documentation)
+
+5. [Contact](#5-contact)
+
+6. [Copyright](#6-copyright)
+
+7. [References](#7-references)
+
+# 1. Introduction
+
+This software package provides implementation of regularized greedy forest
+(RGF) described in [[1]](#7-references).
+
+## 1.1. System Requirements
+
+Executables are provided only for some versions of Windows
```

Comment:
I found the vergion of the executable https://github.com/RGF-team/rgf_python/tree/master/include/rgf/bin is old.
And should we continue to place executable here?

Reply:
> And should we continue to place executable here?

Why not?

>I found the vergion of the executable is old

I'll update them in another PR. There is a lot of work actually here.

Reply:
> Why not?
Is it enough by our github release?

Reply:
We don't actually provide any exe files.

Reply:
*.whl includes exe file. But I know it is hard to understand.

Reply:
Yeah, but these artifacts are for `rgf_python`, not for `RGF`. And of course, no one want to crack wheel file and search for exe file.

Please, don't forget that `rgf_python`, `RGF` and `FastRGF` are separated projects. Any of them could be enhanced variously and at different time.

Reply:
BTW, I want to enhence this section in the future according to our experience in compiling for `rgf_python`. This PR was conceived as just **initial** for converting to `md` format 😃